include/nuttx/fs/fs.h

```diff
@@ -538,7 +538,7 @@ struct filelist
 #ifdef CONFIG_FILE_STREAM
 struct file_struct
 {
-  FAR struct file_struct *fs_next;      /* Pointer to next file stream */
+  sq_entry_t              fs_entry;     /* Entry of file stream */
```

Comment:
should we include queue.h in this file and remove from other source file?

Reply:
Done